scripts/check-ownership/check-ownership.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
```

Comment:
maybe we can put it in a folder `script/npm` so we can use it for other things in the future if we have? or just put the file right under `script`?

Reply:
> maybe we can put it in a folder script/npm so we can use it for other things in the future if we have

Why can't we use it from `scripts/check-ownership/check-ownership.sh` in the future if we need?

I'd prefer to keep `scripts` folder in a more well defined structure:
```
└── scripts
    ├── <action-name> // directoery
    │   └── // one or more files related to the action
    └── <anither-action-name> // directoery
        └── // one or more files related to the action
```

E.g. in next-gen-search project it looks like this: http://tkgit-01.textkernel.net/searchbox/search-ui/tree/dev/scripts
```
└── scripts
    ├── build
    ├── lint
    ├── lint-commits
    ├── lint-styles
    └── test
```

Reply:
Those were just my 2 cents. If we don't have 100s of scripts, it is not really important. I don't think we ever discussed any guidelines about it. The PR is approved, you can merge it.

Reply:
ok, thanks,  I'll do. Btw, what do you think in general about keeping all the scripts under `scripts/<action_name>` directory? Just in order not to keep root pf the project clean and all this stuff more organized?
![image](https://user-images.githubusercontent.com/403250/68658083-14e24480-0535-11ea-8cdd-e78cf3945693.png)